ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ComplianceClusterStatus.tsx

```diff
@@ -35,6 +35,16 @@ function ComplianceClusterStatus({ errors }: ComplianceClusterStatusProps) {
               };
     }
 
+    function getErrorsList(errors: string[]): ReactElement {
+        return (
+            <SimpleList>
```

Comment:
@mtodor Any reason for `SimpleList` instead of (bulleted) `List` element?

Only one use of `SimpleList` element in UI before this, and it was unique, not necessarily in a good way.

https://www.patternfly.org/components/simple-list/design-guidelines
> …or does not require bullets to differentiate one list item from another.

https://www.patternfly.org/components/list/design-guidelines#unordered
> By default, you should use bullets when displaying an unordered list. Use an unordered list when you want to break up related content, but list items are not sequential.

Reply:
@pedrottimark Tnx for the suggestion! To answer, why not `List`? - lack of knowledge. 😄 

I have changed to use a plain list. Bullets take up space in the popup, and they are not significantly improving readability. In most cases, it will be a single error.